src/ReleasableImage.cpp

```diff
@@ -0,0 +1,36 @@
+#include <ZividPython/ReleasableImage.h>
+
+#include <pybind11/pybind11.h>
+
+namespace py = pybind11;
+
+namespace
+{
+    py::buffer_info makeBufferInfo(ZividPython::ReleasableImage &image)
+    {
+        const auto data = image.dataPtr();
+
+        return py::buffer_info{
+            const_cast<Zivid::RGBA8 *>(
+                data), // TODO: Const casting this until https://github.com/pybind/pybind11/issues/1993 has been solved
```

Comment:
This is now merged. The fix is now to simply remove the const cast once pybind11 is updated.

Reply:
Updated the comment